gui/src/main/java/io/bitsquare/gui/util/MakeSound.java

```diff
@@ -0,0 +1,75 @@
+package io.bitsquare.gui.util;
+
+import java.io.File;
+import java.io.IOException;
+
+import javax.sound.sampled.AudioFormat;
+import javax.sound.sampled.AudioInputStream;
+import javax.sound.sampled.AudioSystem;
+import javax.sound.sampled.DataLine;
+import javax.sound.sampled.LineUnavailableException;
+import javax.sound.sampled.SourceDataLine;
+
+public class MakeSound {
+
+    private final int BUFFER_SIZE = 128000;
```

Comment:
where does this number come from? You don't have to change it but maybe a comment would be nice to show how it was calculated.